.travis.yml

```diff
@@ -12,6 +12,7 @@ install:
   - sudo npm install -g npm
   - sudo npm install -g azure-cli
   - go get -u github.com/golang/lint/golint
+  - go get -u github.com/ahmetalpbalkan/govvv
```

Comment:
shouldn't you also update the actual build script to use `govvv` instead of just `go`?